saleor/graphql/checkout/mutations/utils.py

```diff
@@ -414,3 +437,12 @@ def find_line_id_when_variant_parameter_used(variant_id, lines_info):
         )
 
     return str(line_info[0].line.id)
+
+
+def find_variant_id_when_line_parameter_used(line_id, lines_info):
+    """Return variant id by using provided lineId parameter."""
```

Comment:
Could we add info that lineId is DB id or global id?

Reply:
I will add typing to this function